src/sync-signals-with-metadata.py

```diff
@@ -0,0 +1,167 @@
+import logging
+import os
+from datetime import datetime
+
+import requests
+import sqlalchemy
+from sqlalchemy import create_engine, update
+from sqlalchemy.engine import Engine
+from sqlalchemy.orm import DeclarativeBase, sessionmaker
+
+logger = logging.getLogger(__name__)
+
+SQLALCHEMY_DATABASE_URI = os.environ.get(
+    "SQLALCHEMY_DATABASE_URI",
+    "mysql+mysqlconnector://root:ROOT_PASSWORD@localhost:3306/mysql_database",
+)
+engine: Engine = create_engine(
+    SQLALCHEMY_DATABASE_URI, execution_options={"engine_id": "default"}
+)
+Session = sessionmaker(bind=engine)
+
+
+COVID_CAST_META_URL = os.environ.get(
+    "COVID_CAST_META_URL", "https://api.delphi.cmu.edu/epidata/covidcast/meta"
+)
+
+
+class Base(DeclarativeBase):
+    pass
+
+
+class Signal(Base):
+
+    __tablename__ = "signals_signal"
+
+    id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True)
+    name = sqlalchemy.Column(sqlalchemy.String)
+    display_name = sqlalchemy.Column(sqlalchemy.String)
+    active = sqlalchemy.Column(sqlalchemy.Boolean)
+    short_description = sqlalchemy.Column(sqlalchemy.String)
+    description = sqlalchemy.Column(sqlalchemy.String)
+    format_type = sqlalchemy.Column(sqlalchemy.String)
+    time_type = sqlalchemy.Column(sqlalchemy.String)
+    time_label = sqlalchemy.Column(sqlalchemy.String)
+    is_smoothed = sqlalchemy.Column(sqlalchemy.Boolean)
+    is_weighted = sqlalchemy.Column(sqlalchemy.Boolean)
+    is_cumulative = sqlalchemy.Column(sqlalchemy.Boolean)
+    has_stderr = sqlalchemy.Column(sqlalchemy.Boolean)
+    has_sample_size = sqlalchemy.Column(sqlalchemy.Boolean)
+    high_values_are = sqlalchemy.Column(sqlalchemy.String)
+    base_id = sqlalchemy.Column(sqlalchemy.Integer)
+    category_id = sqlalchemy.Column(sqlalchemy.Integer)
+    source_id = sqlalchemy.Column(sqlalchemy.Integer)
+    created = sqlalchemy.Column(sqlalchemy.DateTime)
+    modified = sqlalchemy.Column(sqlalchemy.DateTime)
+    last_updated = sqlalchemy.Column(sqlalchemy.Date)
+    age_breakdown = sqlalchemy.Column(sqlalchemy.String)
+    data_censoring = sqlalchemy.Column(sqlalchemy.Text)
+    gender_breakdown = sqlalchemy.Column(sqlalchemy.Integer)
+    missingness = sqlalchemy.Column(sqlalchemy.Text)
+    race_breakdown = sqlalchemy.Column(sqlalchemy.Integer)
+    reporting_cadence = sqlalchemy.Column(sqlalchemy.String)
+    restrictions = sqlalchemy.Column(sqlalchemy.Text)
+    severenity_pyramid_rungs = sqlalchemy.Column(sqlalchemy.String)
```

Comment:
```suggestion
    severity_pyramid_rungs = sqlalchemy.Column(sqlalchemy.String)
```
you were talking about fixing this... is this something we can do now, or should we wait so we can change the spelling in other places at the same time?

Reply:
@melange396 Oh no that happened again. Sorry for that 😢 
BUT, we may not want to apply this change right now because I've not merged PR with fix in the signal-documentation repo, so this field actually exists in signal-documentation's database 🤡